packages/autotask-client/src/utils.ts

```diff
@@ -34,3 +41,38 @@ export function validatePath(path: string): void {
     throw new Error(`path ${path} does not exist`);
   }
 }
+
+export async function tailLogsFor(client: AutotaskClient, autotaskId: string) {
+  try {
+    validateId(autotaskId);
+    console.warn(`\nPolling latest runs of autotask '${autotaskId}'...\n`);
+    // Poll autotask runs every 2 seconds and if there are new runs, get run details and print them out.
+    let lastRun: AutotaskRunListItemResponse | undefined;
+    while (true) {
+      const newRuns = await client.listAutotaskRuns(autotaskId);
+      // If cached last run id has changed
+      if (newRuns.items[0]?.autotaskRunId !== lastRun?.autotaskRunId) {
+        lastRun = newRuns.items[0]; // cache new last run to avoid duplicates.
+        const status = lastRun.status as AutotaskRunStatus;
+        if (status === 'pending') {
+          lastRun = undefined; // clean up so we can check it again on the next poll.
+        } else if (status === 'error') {
+          const runDetails = (await client.getAutotaskRun(lastRun.autotaskRunId)) as AutotaskRunErrorData;
+          console.log(`\nError: ${runDetails.message}`);
+          runDetails.decodedLogs ? console.log(`\n${runDetails.decodedLogs}`) : console.log(`No logs available.`);
+        } else if (status === 'success') {
+          const runDetails = (await client.getAutotaskRun(lastRun.autotaskRunId)) as AutotaskRunSuccessData;
+          console.log(`\n${runDetails.decodedLogs}`);
+        } else if (status === 'throttled') {
+          console.warn(
+            `\nThis autotask run was canceled since the hourly run capacity for your account has been exceeded. Contact us at defender-support@openzeppelin.com for additional capacity.`,
+          );
+        }
+      }
+
+      await new Promise((resolve) => setTimeout(resolve, 2000));
+    }
+  } catch (e) {
+    throw e;
+  }
```

Comment:
To be annoying, you don't really need a try/catch here if you're just rethrowing. But that's a nitpick.